src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1

```diff
@@ -74,7 +74,36 @@ CmdletsToExport = '*'
 VariablesToExport = '*'  
   
 # Aliases to export from this module  
-AliasesToExport = @()
+AliasesToExport = @(
```

Comment:
Is this needed? Shouldn't this happen automatically, if our intent is to expose all aliases